stim/stim.go

```diff
@@ -3,48 +3,48 @@ package stim
 import (
 	"os"
 	"os/user"
-	"strings"
-	"sync"
+	"path/filepath"
 
 	"github.com/PremiereGlobal/stim/pkg/stimlog"
 	"github.com/PremiereGlobal/stim/pkg/vault"
+	"github.com/mitchellh/go-homedir"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+var defaultStimConfigFilePath string
+var version string
+
+func init() {
+	home, err := homedir.Dir()
+	if err != nil {
+		defaultStimConfigFilePath = filepath.Join(os.TempDir(), ".stim", "config.yaml")
```

Comment:
Why would this happen? When would this code run and there isn't a home dir? os.TempDir()

Reply:
Yeah, say a badly setup build machine or something like that with no home env, we need to do something thats the best idea I had.